src/pages/home/report/ContextMenu/ContextMenuActions.tsx

```diff
@@ -354,7 +354,15 @@ const ContextMenuActions: ContextMenuAction[] = [
                     const modifyExpenseMessage = ModifiedExpenseMessage.getForReportAction(reportID, reportAction);
                     Clipboard.setString(modifyExpenseMessage);
                 } else if (ReportActionsUtils.isMoneyRequestAction(reportAction)) {
-                    const displayMessage = ReportUtils.getIOUReportActionDisplayMessage(reportAction);
+                    const displayMessage = ReportUtils.getReportPreviewMessage(
```

Comment:
I see there are many differences between `getIOUReportActionDisplayMessage` and `getReportPreviewMessage` functions. Using `getReportPreviewMessage` will change other behaviors (not relevant to this issue)
This is an example

https://github.com/Expensify/App/assets/141406735/509238fe-e0a5-4cd8-97df-e23ddd9089b0

Reply:
@dukenv0307 Also remove getIOUReportActionDisplayMessage function

Reply:
What happened here? `getReportPreviewMessage` retrieves the message for the report preview. We've been displaying the wrong message for IOU actions for a while, but the messages displayed by `getIOUReportActionDisplayMessage` are the right ones. If that causes other bugs, we need to address them, but not revert back to the wrong messages. Was there a discussion about this on Slack?